channeldb/channel.go

```diff
@@ -3515,7 +3468,8 @@ func fetchChanInfo(chanBucket kvdb.RBucket, channel *OpenChannel) error {
 		&channel.chanStatus, &channel.FundingBroadcastHeight,
 		&channel.NumConfsRequired, &channel.ChannelFlags,
 		&channel.IdentityPub, &channel.Capacity, &channel.TotalMSatSent,
-		&channel.TotalMSatReceived,
+		&channel.TotalMSatReceived, &channel.InitialLocalBalance,
```

Comment:
Reflecting back, I think we should've (not too late though?) have added these as TLV fields below. This seems to be the source of https://github.com/lightningnetwork/lnd/issues/6548 as the historical channel bucket wasn't also migrated, and we have this assumption that these fields exist in line.

Reply:
This was subtle - I guess that's why this bug didn't happen before since TLV fields were used instead